### ISSUE TITLE
Popful Mail - Letting opening cutscene run drops to a black screen on first cut

### ISSUE THREAD

**Issue by YLFAndy** [Mazamars312/openfpga-pcengine-cd#8]
Description:
After choosing Begin at the start screen, the cutscene that plays next drops to black and plays audio only after first cut.

Repro Steps:
1. Load Popful Mail via an appropriate JP bios (any other bios will result in a warning message from one of the characters).
2. Press Run after the first cutscene begins (note, there are some cramdots at the top of the screen, or possibly an off by one px error)
3. Select Begin and press Run
4. Let the next cutscene play out

Result:
At the first cut, the screen dips to black and the audio continues. Run can be pressed to skip.

Frequency: 100%
Recoverable: Yes

Expected:
The scene should cut to the hero running, then having a weapon thrown at her. The same image works on MiSTer. 

                                                                                                                                                                                    
[popfulbug.zip](https://github.com/Mazamars312/openfpga-pcengine-cd/files/10671441/popfulbug.zip)

**Comment by YLFAndy** [comment]
Confirmed the behavior persists in the new build.

**Comment by YLFAndy** [comment]
Confirmed the behavior persists with the correct artifacts for the new build :)

**Comment by Mazamars312** [comment]
Yep I can see this. I need to find what is going on here. Does this game require both VDP's? AS right now we where not able to fit both on right now

**Comment by YLFAndy** [comment]
I'd have to debug on MiSTer. I'll be back in the office with my dev unit tomorrow and can see what calls it makes at that moment, if it dumps that.

**Comment by Mazamars312** [comment]
Sweet thank you very much.

**Comment by YLFAndy** [comment]
I'm not super familiar with FPGA debugging. It looks like the debugger exists only on the host level and doesn't monitor anything inside the core. Here's my log from the Pocket. 

The title screen behavior makes me think your assertion is probably correct though. The options flicker badly on Pocket and are solid on Duo. The flicker seems to be every other frame, which fits with missing VDP. 


[Mazamars312.PC Engine CD_20230207_092931.zip](https://github.com/Mazamars312/openfpga-pcengine-cd/files/10677880/Mazamars312.PC.Engine.CD_20230207_092931.zip)

**Comment by YLFAndy** [comment]
Popful is a > 27 bin disc. I'm wondering if this is related to https://github.com/Mazamars312/openfpga-pcengine-cd/issues/16

Since asset streaming seems to be offset a little bit with merged bins.

**Comment by YLFAndy** [comment]
0.1.7 - audio is now very jittery, animation still fails to load (VDP2 still, probably). However, the lag of the audio on first animation load appears to be better (that is, it starts closer to the point when it's supposed to start). Let me know if I can provide any logs or captures.
